packages/foam-vscode/src/features/link-completion.ts

```diff
@@ -199,6 +199,21 @@ export class CompletionProvider
       item.commitCharacters = linkCommitCharacters;
       return item;
     });
+    resources.push(
```

Comment:
have you considered creating a `const aliases = ...`? It's more in line with the style of the function, it doesn't modify the `resources` variable (which is technically defined as `const`, so it can also be misleading) and still makes it easy to place them where we want in the final list of suggestions.

Also, have you considered looping through resources only once and returning both `resources` and `aliases`?